newt/image/image.go

```diff
@@ -46,6 +46,10 @@ import (
 	"mynewt.apache.org/newt/util"
 )
 
+// Set this to enable RSA-PSS for RSA signatures, instead of PKCS#1
+// v1.5.  Eventually, this should be the default.
+var UseRsaPss = false
```

Comment:
Ah, I forgot that there was a layer of indirection added between the CLI and image signing for split images. Reaching through that will be a bit cumbersome.